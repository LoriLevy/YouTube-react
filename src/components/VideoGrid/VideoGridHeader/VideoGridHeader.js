import React from 'react';
import './VideoGridHeader.scss';

export function VideoGridHeader(props) {
  return (
    <div className='video-grid-header'>
      <h4 className='video-category-title'>{props.title}</h4>
    </div>
  );
}
