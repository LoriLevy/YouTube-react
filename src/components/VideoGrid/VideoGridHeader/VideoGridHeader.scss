/* file: src/components/VideoGrid/VideoGridHeader/VideoGridHeader.scss */
.video-grid-header {
  padding-bottom: 24px;
  padding-top: 14px;
  
  .video-category-title {
    font-size: 1.2rem;
    font-weight: bold;
  }
}
