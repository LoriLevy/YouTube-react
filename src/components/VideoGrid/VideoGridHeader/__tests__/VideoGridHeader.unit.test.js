import React from "react";
import { shallow } from "enzyme";
import { VideoGridHeader } from "../VideoGridHeader";

describe("VideoGridHeader", () => {
  test("renders without props", () => {
    const wrapper = shallow(<VideoGridHeader />);
    expect(wrapper).toMatchSnapshot();
  });
  test("renders with empty string title", () => {
    const wrapper = shallow(<VideoGridHeader title="" />);
    expect(wrapper).toMatchSnapshot();
  });
  test("renders with a title", () => {
    const wrapper = shallow(<VideoGridHeader title="Antique Dolls" />);
    expect(wrapper).toMatchSnapshot();
  });
});
