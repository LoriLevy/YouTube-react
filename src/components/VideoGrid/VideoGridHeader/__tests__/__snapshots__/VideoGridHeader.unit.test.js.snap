// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoGridHeader renders with a title 1`] = `
<div
  className="video-grid-header"
>
  <h4
    className="video-category-title"
  >
    Antique Dolls
  </h4>
</div>
`;

exports[`VideoGridHeader renders with empty string title 1`] = `
<div
  className="video-grid-header"
>
  <h4
    className="video-category-title"
  />
</div>
`;

exports[`VideoGridHeader renders without props 1`] = `
<div
  className="video-grid-header"
>
  <h4
    className="video-category-title"
  />
</div>
`;
